spec/models/user_spec.rb

```diff
@@ -1,5 +1,75 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "validations" do
+    context "invalid attributes" do
+      it { should have_secure_password }
+      it { should validate_presence_of(:first_name) }
+      it { should validate_presence_of(:last_name) }
+      it { should validate_presence_of(:email) }
+      it { should validate_presence_of(:username) }
+      it { should validate_uniqueness_of(:username) }
+      it { should validate_presence_of(:phone_number) }
+    end
+
+    context "valid attributes" do
+      it "is valid with a first name, last name, username, email, phone number and password" do
+        user = User.new(
+                        first_name: "Mason",
+                        last_name: "Tom",
+                        email: "mason@tom.com",
+                        username: "ToMas",
+                        phone_number: "303-111-2222",
+                        password: "Password"
+                        )
+
+        expect(user).to be_valid
+      end
+    end
+  end
+
+  context "relationships" do
+    xit "belongs to ..." do
+      user = User.new(
+                      first_name: "Mason",
+                      last_name: "Tom",
+                      email: "mason@tom.com",
+                      username: "ToMas",
+                      phone_number: "303-111-2222",
+                      password: "Password"
+                      )
+
+      expect(user).to resond_to(:model)
```

Comment:
You should correct the type in `respond_to` here and in the pastes below cc: @MasonHolland , @TomSoucie